cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -98,7 +98,37 @@ macro(swift_common_standalone_build_config_llvm product)
   endif()
 
   if(LLVM_ENABLE_ZLIB)
-   find_package(ZLIB REQUIRED)
+    find_package(ZLIB REQUIRED)
+  endif()
+
+  # Work around a bug in the swift-driver that causes the swift-driver to not be
+  # able to accept .tbd files when linking without passing in the .tbd file with
+  # a -Xlinker flag.
+  #
+  # Both clang and swiftc can accept an -Xlinker flag so we use that to pass the
+  # value.
+  if (APPLE)
+    get_target_property(LLVMSUPPORT_INTERFACE_LINK_LIBRARIES LLVMSupport INTERFACE_LINK_LIBRARIES)
+    set(new_libraries)
+    foreach(lib ${LLVMSUPPORT_INTERFACE_LINK_LIBRARIES})
+      if (TARGET ${lib})
+        list(APPEND new_libraries "-Xlinker;$<TARGET_FILE:${lib}>")
```

Comment:
You can't do this this way.  This will only work for a single library.  You need to use this with `SHELL:` to ensure that the `-Xlinker` doesn't get dropped.